spec/models/journaled/writer_spec.rb

```diff
@@ -122,17 +117,25 @@
 
           it 'defaults to the global default' do
             expect { subject.journal! }.to change {
-              Delayed::Job.where('handler like ?', '%Journaled::Delivery%').where(priority: 999).count
+              if Rails::VERSION::MAJOR < 6
+                enqueued_jobs.select { |j| j[:job] == Journaled::DeliveryJob }.count
```

Comment:
is there any way to assert on priority here?

Reply:
ActiveJob on Rails 5.2 doesn't actually support priority at all. So, no. 😞

Reply:
I updated the Readme to reflect this!